custom/icds_reports/tasks.py

```diff
@@ -693,7 +695,25 @@ def prepare_excel_reports(config, aggregation_level, include_test, beta, locatio
             )
         else:
             cache_key = create_excel_file(excel_data, data_type, file_format)
-    if indicator != AWW_INCENTIVE_REPORT:
+    elif indicator == LS_REPORT_EXPORT:
+        data_type = 'Lady_Supervisor'
+        config['aggregation_level'] = 4  # this report is explicitly not aggregated
```

Comment:
Can you make this comment clearer. First glance  looks that the data is not aggregated but the code will do that later. I think something like 'this report is not to be aggregated' or something similar will be good for people who does not know the spec of this report

Reply:
I am going to change it to: `this report on all levels shows data (row) per sector`

Reply:
Sounds good